First draft volunteer newsletter

nabramow opened the pull request:
Please leave your feedback, suggest changes, and share any shoutouts I missed as I wasn't here most of the year and don't want to leave anyone out!

nabramow committed:
First draft volunteer newsletter

nabramow committed:
Merge branch 'develop' into na/volunteer-newsletter

nabramow committed:
Forgot to save

nabramow commented:
@spreeni I know you did the user survey and I'd love to include that here I just didn't know how to describe it so if you want to add a blip or send me a ticket that describes it please do!

nabramow commented:
@Krishnag09 Don't want to leave you out! I think you did a lot before I started and I don't always get your updates in the meeting so send me a blurb of what you've done or the tickets you worked on and I'll add it!

nabramow commented on docs/meetings/newsletters/2024-01-04_volunteer_newsletter.md:
```diff
@@ -0,0 +1,87 @@
+* 🗺️ **Map Redesign**: Design and development support needed! Check out the [map redesign ticket](https://github.com/Couchers-org/couchers/issues).
+* 🌐 **Public Events**: We’re adding a feature to create shorter, public versions of events so they can be shared with non-Couchers. [More info in the epic.](https://github.com/Couchers-org/couchers/issues/5339)
+* ⬆️ **Frontend Package Upgrades**: Help us stay current by upgrading to React v19, Next.js v14, Material-UI v6, and react-query v3, among others.
+* 🚫 **User Blocking & Reporting**: We’re working on implementing user-blocking functionality, expanding reporting to comments and implementation of our private feedback flow (designs ready)
+
```
Is there a ticket for this? I'd love to link it but can't find anything @jesseallhands

nabramow replied:
Added it!

nabramow commented:
Also considering maybe a welcome to our new volunteers section and tagging everyone who has merged something in the last months, but trying to decide the boundary there is tricky haha.

nabramow committed:
Add links for private feedback and reference flow tickets

spreeni commented:
> @spreeni I know you did the user survey and I'd love to include that here I just didn't know how to describe it so if you want to add a blip or send me a ticket that describes it please do!

**Couchers 2024 User Survey**
We conducted a survey among all Couchers.org users to gain an insight into the main issues and priorities of our users. Improvements to the map search and filtering of inactive/incomplete profiles were the most requested feautures in the survey.

nabramow committed:
Add Yannic's blurb

aapeliv commented:
As I briefly mentioned on the weekly call, I think trying to shape this to be more actionable would be good. We should reduce the hurdle to contribute as much as possible. So would be good to add a call to action with clear instructions: "pop back in to slack", "join our weekly meeting at [time] at [location]", etc. And where possible good to add links to github issues/PRs and/or define clearly the task and basically make it "go here and start on this" without any intervention from us.

nabramow committed:
Merge branch 'develop' into na/volunteer-newsletter

nabramow committed:
Make it easier to find how to get involved

nabramow commented:
> As I briefly mentioned on the weekly call, I think trying to shape this to be more actionable would be good. We should reduce the hurdle to contribute as much as possible. So would be good to add a call to action with clear instructions: "pop back in to slack", "join our weekly meeting at [time] at [location]", etc. And where possible good to add links to github issues/PRs and/or define clearly the task and basically make it "go here and start on this" without any intervention from us.

Okay @aapeliv I added a "How to get involved" section, added a table of contents and anchor links so it's easier to click around, linked the "how to get involved" after the help wanted sections, and made the engineering help wanted section more ticket focused.

Let me know what you think. If you want more edits let me know more specifically what you're thinking.

Super-Programmer-Navi reviewed (dismissed):
Looks good to me, very upbeat!

aapeliv commented on docs/meetings/newsletters/2024-01-04_volunteer_newsletter.md:
```diff
@@ -0,0 +1,116 @@
+## 🎉 Welcome to Our First Edition!
+
+Hello, amazing Couchers volunteers!  
+If you’re reading this, you’ve volunteered with Couchers in some capacity. With the New Year underway, we have some resolutions of our own and big goals going forward. We’ve had a surge of volunteer activity, so we wanted to share with you what we’ve been working on, what we plan to do, and what we could use help with in case this stirs any FOMO 😉.
```
If you’re reading this, you’ve volunteered with Couchers in some capacity in the past or are doing so now. With the New Year underway, we have some resolutions of our own and big goals going forward. We’ve had a surge of volunteer activity, so we wanted to share with you what we’ve been working on, as well as what we plan to do, and what we could use your help with in case this stirs any FOMO 😉.

nabramow replied:
Your text you wrote here is the same as mine or am I crazy?

aapeliv commented on docs/meetings/newsletters/2024-01-04_volunteer_newsletter.md:
```diff
@@ -0,0 +1,116 @@
+
+Hello, amazing Couchers volunteers!  
+If you’re reading this, you’ve volunteered with Couchers in some capacity. With the New Year underway, we have some resolutions of our own and big goals going forward. We’ve had a surge of volunteer activity, so we wanted to share with you what we’ve been working on, what we plan to do, and what we could use help with in case this stirs any FOMO 😉.
+
```
We'll be sending out this volunteer newsletter intermittently, maybe once a quarter or so. It probably won't be very frequent since we are a bit lousy at writing newsletters (if you want to help, let us know). We appreciate all of your amazing contributions and want to share some more details with you and let you know of any opportunities to jump back in.

That said, if you don't want to receive these newsletters, please click on the big button below and we'll unsubscribe you, no dramas!

[[UNSUB BUTTON]]

aapeliv commented on docs/meetings/newsletters/2024-01-04_volunteer_newsletter.md:
```diff
@@ -0,0 +1,116 @@
+* Tag @nabramow, @aapeliv or @jesseallhands in a ticket you'd like to pick up and assign yourself to it. Check out our [Couchers Engineering Task Board](https://github.com/Couchers-org/couchers/projects) for what’s in progress or ready to tackle. See Help Wanted for [Engineering](#-help-wanted-engineering), [Design](#️-help-wanted-design) and [Operations](#️-help-wanted-operations)
+
+
+## 💻 [Engineering Accomplishments](#-engineering-accomplishments)
+
```
# Engineering Updates

Since the last public newsletter went out, we've put our heads down and mostly been grinding out feature updates and bugfixes for the web platform. We also have an ongoing effort to build a mobile app in React Native. Here's a short list of some engineering accomplishments:

aapeliv commented on docs/meetings/newsletters/2024-01-04_volunteer_newsletter.md:
```diff
@@ -0,0 +1,116 @@
+
+* ❓ **Couchers 2024 User Survey**: We conducted a survey among all Couchers.org users to gain insight into the main issues and priorities of our users. Improvements to the map search and filtering of inactive/incomplete profiles were the most requested feautures in the survey. Shoutout to [Yannic](https://github.com/spreeni) who had the main role in this!
+
+### ✍️ [Help Wanted: Design](#️-help-wanted-design)
```
I'll have to find my little list of other things on my mind.

Need to add:

* Couchers Engineering logo
* Couchers stickers
* Mail verification postcard!

nabramow replied:
Do you want these in the Help Wanted: Design section?

So we want to design an Couchers Engineering specific logo?

aapeliv commented on docs/meetings/newsletters/2024-01-04_volunteer_newsletter.md:
```diff
@@ -0,0 +1,116 @@
+
+We could use a hand with:
+
+* 🌍 **Translations**: We’re behind on translating the site into other languages. If you’re interested in managing the translation process or helping with translations, let us know!  
```
divide into 

* Translation lead organizer: with the new feature updates and changes to the platform, we've fallen behind on translating the app. If you're interested in helping organize this effort to translate the new parts, please get in touch!
* Translators: just want to help us translate the app into your language? Please get in touch!

I'd be good if we had a shortlist of key languages.

Also add

* Support people

aapeliv commented on docs/meetings/newsletters/2024-01-04_volunteer_newsletter.md:
```diff
@@ -0,0 +1,116 @@
+4. 📱 **Native App Design**: We need designs for the standalone mobile app.
+
+---
+
```
Please add a section:

# Help Wanted: Marketing, Branding, and Outreach

* Newsletters: these are some of the most prominent ways in which we get the word out, and we'd like to send one out four times a year to communicate latest updates and our latest strategy. Please come help us write some!
* Website rebranding: we want to change our branding from being simply an alternative to CouchSurfing.com™ into standing on our own feet as the safest and coolest couch surfing platform. Come help us rewrite and redesign our landing pages for this new angle!
* Marketing: we are looking for someone to help us ...
* Writers: we want to create more newbie-friendly content to bring more people into couch surfing, including a section on the website for "what is couch surfing" to explain to those who've never thought about it what it is, come help in this effort!

jesseallhands replied:
I think before we start recruiting marketing folks, we probably should recruit a lead marketing person who then can help us recruit and lead other marketing folks. Otherwise we will be recruiting people without a leader and may cause volunteers to drop off because they are lost/don't have guidance.

jesseallhands replied:
For example, having 5 or 6 marking folk willing to do tasks but not lead/do strategic thinking could become more of a burden for us than help us - and also disenchant the volunteers

nabramow replied:
Okay I took all these comments plus some of my own liberties ;-)

aapeliv reviewed (commented):
sorry, i feel like i dumped way too much stuff here

nabramow committed:
Edits based on comments

nabramow committed:
Another update

nabramow commented:
Okay @aapeliv @jesseallhands changes made!

spreeni commented:
Great work, very concise and readable while also sounding genuine! 👍 For me, all comments look to be integrated as well.

nabramow commented:
Closing here as it was officially sent on listmonk!
